flexget/plugins/cli/series.py

```diff
@@ -282,8 +284,14 @@ def register_parser_arguments():
     order.add_argument('--descending', dest='order', action='store_true', help='Sort in descending order')
     order.add_argument('--ascending', dest='order', action='store_false', help='Sort in ascending order')
 
-    subparsers.add_parser('show', parents=[series_parser, table_parser],
-                          help='Show the releases FlexGet has seen for a given series ')
+    show_parser = subparsers.add_parser('show', parents=[series_parser, table_parser],
+                          help='Show the releases FlexGet has seen for a given series')
+    show_parser.add_argument('--sort-by', choices=('seen', 'entity'), default='seen',
```

Comment:
`age` is a good idea. I was trying to avoid using underscored suffixes in an attribute as I think it makes it easier to remember and use, but I can change it if you feel strongly about it.